sensor_msgs/Image input_image
---
Box[] faces